scripts/pkg/build.js

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env node
+
+// Node.js v8+ only
+
+'use strict';
+
+require('essentials');
+
+const path = require('path');
+const spawn = require('child-process-ext/spawn');
+const BbPromise = require('bluebird');
+const fse = require('fs-extra');
+
+BbPromise.promisifyAll(fse);
+
+const serverlessPath = path.join(__dirname, '../..');
+const spawnOptions = { cwd: serverlessPath, stdio: 'inherit' };
+
+(async () => {
+  // To bundle npm with a binary we need to install it
+  process.stdout.write('Install npm\n');
+  // Hard code npm version to one that comes with lastest Node.js
+  // It's due to fact that npm tends to issue buggy releases
+  // Node.js confirms on given version before including it within its bundle
+  // Version mappings reference: https://nodejs.org/en/download/releases/
+  await spawn('npm', ['install', '--no-save', 'npm@6.12.1'], spawnOptions);
+
+  try {
+    process.stdout.write('Build binaries\n');
+    await spawn(
+      'node',
+      [
+        './node_modules/.bin/pkg',
+        '-c',
+        'scripts/pkg/config.js',
+        '--targets',
+        'node12-linux-x64,node12-mac-x64,node12-win-x64',
```

Comment:
Not sure where to track that but we need to ensure that we update this section once new LTS versions are available.

Reply:
Yes, same way we should add new Travis job when new Node.js version is introduced.

Still I believe those changes are handled naturally, and we do not have specific document to remind us of that (?)

Reply:
Yes, we do not have that right now. Maybe we can introduce something like that in a separate PR.

I'm fine with leaving it as is right now, just want to ensure that we're aware of this.

Reply:
Ok, still it's not perfectly clear for me what you're proposing (?)

Creation of document that should list tasks that should be done every time new Node.js version is introduced?

Reply:
Yes, exactly. Some central place we can consult.